src/screens/Classes/ClassTabs.js

```diff
@@ -0,0 +1,71 @@
+
+import React , { Component } from "react";
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/Airbenders-490/AirbendersFE/pullRequest?prid=8258429)
```suggestion
import React , { Component } from 'react';
```